lua/telescope/_extensions/file_browser/picker.lua

```diff
@@ -48,6 +73,7 @@ local fb_picker = {}
 ---@field grouped boolean: group initial sorting by directories and then files; uses plenary.scandir (default: false)
 ---@field files boolean: start in file (true) or folder (false) browser (default: true)
 ---@field add_dirs boolean: whether the file browser shows folders (default: true)
+---@field select_buffer boolean: whether to select current buffer at start if possible (default: false)
```

Comment:
Let's make clear this entails `hidden=true` if `select_buffer=true` and the current buffer is a hidden file